plugin/symbols.py

```diff
@@ -15,29 +16,23 @@
 SUPPRESS_INPUT_SETTING_KEY = 'lsp_suppress_input'
 
 
-def unpack_lsp_kind(kind: int) -> Tuple[int, str, str, str]:
-    if 1 <= kind <= len(SYMBOL_KINDS):
-        return SYMBOL_KINDS[kind - 1]
-    return sublime.KIND_ID_AMBIGUOUS, "?", "???", "comment"
+def unpack_lsp_kind(kind: int) -> Tuple[int, str, str]:
+    return SYMBOL_KINDS[kind] if kind in SYMBOL_KINDS else (sublime.KIND_ID_AMBIGUOUS, "?", "???")
```

Comment:
Would it be possible to avoid the double dict lookup for the key here? We do a lookup for the `item_kind` key in `kind in SYMBOL_KINDS` as test, and then do the same lookup again in `SYMBOL_KINDS[kind]`. Perhaps use something like `st_kind = SYMBOL_KINDS.get(kind)` and test for a `None` value, which is just **1** lookup in the dict.